server/test/server.test.js

```diff
@@ -105,9 +105,20 @@ describe("Testing Registration and Login", () => {
 // Tests for testing tweet backend functionality
 
 describe("Testing backend Tweet logic", () => {
-	test("Logging in, and Creating a tweet", async () => {
```

Comment:
Maybe also create a test with an existing user?

Reply:
Additional Tests will be added on later, including one that you mentioned.